### ISSUE TITLE
failing test - tests/unit/test__http.py:TestConnection.test_extra_headers

### ISSUE THREAD

**Issue by rvandegrift** [googleapis/python-resource-manager#9]
`test_extra_headers` is failing due to the `timeout` parameter not matching:

```
$ python3 -m unittest
...F..............................
======================================================================
FAIL: test_extra_headers (tests.unit.test__http.TestConnection)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ross/src/googleapis/python-resource-manager/tests/unit/test__http.py", line 81, in test_extra_headers
    timeout=None,
  File "/home/ross/.venvs/resource-manager/lib/python3.7/site-packages/mock/mock.py", line 926, in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
  File "/home/ross/.venvs/resource-manager/lib/python3.7/site-packages/mock/mock.py", line 913, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: expected call not found.
Expected: request(data='req-data-boring', headers={'Accept-Encoding': 'gzip', 'X-Goog-API-Client': 'gl-python/3.7.3 gax/1.16.0 gapic/0.30.0 gccl/0.30.0', 'User-Agent': 'gl-python/3.7.3 gax/1.16.0 gapic/0.30.0 gccl/0.30.0'}, method='GET', url='https://cloudresourcemanager.googleapis.com/v1beta1/rainbow', timeout=None)
Actual: request(url='https://cloudresourcemanager.googleapis.com/v1beta1/rainbow', method='GET', headers={'Accept-Encoding': 'gzip', 'X-Goog-API-Client': 'gl-python/3.7.3 gax/1.16.0 gapic/0.30.0 gccl/0.30.0', 'User-Agent': 'gl-python/3.7.3 gax/1.16.0 gapic/0.30.0 gccl/0.30.0'}, data='req-data-boring', timeout=60)

----------------------------------------------------------------------
Ran 34 tests in 0.022s

FAILED (failures=1)
```

**Comment by busunkim96** [comment]
I believe this will be resolved one #4 is merged. Thanks for filing an issue!

**Comment by tseaver** [comment]
Via #4.
